### ISSUE TITLE
Update bundle and build script

### ISSUE THREAD

**Issue by DecampsRenan** [BearStudio/start-ui-native#24]
The `scripts/android-bundle.sh` and `scripts/android-build.sh` are both doing this command: `react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output app/src/main/assets/index.android.bundle --assets-dest app/src/main/res`

This command is no longer mandatory to be able to bundle or build an android app (see the yellow note here: https://reactnative.dev/docs/signed-apk-android#generating-the-release-aab)

Basically the gradle scripts will handle that for us (through the `assembleRelease` and `bundleRelease` tasks)
